### PR TITLE
more parameters for cli tool

### PR DESCRIPTION
**Pull request by sebix**
* new: --subject
* new: --time-interval
* fixed: --text

### CODE REVIEW

File: intelmq/bin/intelmqcli.py [hunk 1]
```diff
@@ -131,15 +135,19 @@ def init(self):
             self.logger.info("All taxonomies: " + ", ".join(taxonomies))
             for taxonomy in taxonomies:
                 self.logger.info('Handling taxonomy {!r}.'.format(taxonomy))
-                if taxonomy not in lib.SUBJECT or lib.SUBJECT[taxonomy] is None:
+                if (taxonomy not in lib.SUBJECT or lib.SUBJECT[taxonomy] is None) and not self.subject:
```
Comment (aaronkaplan) [inline intelmq/bin/intelmqcli.py:138]: as discussed today: should we be case insensitive here?

**Review by aaronkaplan — APPROVED** [review]
looks ok. Fine for me. Please have @otmarlendl review it as well

File: intelmq/bin/intelmqcli.py (hunk 1 above)
Comment (ghost) [reply intelmq/bin/intelmqcli.py:138]: With certat/intelmq#110 merged, lower case values will be enforced for taxonomies.

**Comment by aaronkaplan** [comment]
merging after discussion

**Pull request merged**